.env

```diff
@@ -15,6 +15,18 @@ ODOO_PORT=8069
 ODOO_ATOMFEED_USER=admin
 ODOO_ATOMFEED_PASSWORD=admin
 
+# Mail Config Properties Environment Variables
+MAIL_TRANSPORT_PROTOCOL=
+MAIL_SMTP_HOST=
+MAIL_SMTP_PORT=
+MAIL_SMTP_AUTH=
+MAIL_SMTP_STARTTLS_ENABLE=
+MAIL_SMTP_SSL_ENABLE=
+MAIL_DEBUG=
```

Comment:
You can pass on default values from here. Just leave secrets like HOST, PORT, USER, PASSWORD empty. Others you can add default / recommended values

Reply:
Added @mohan-13